packages/core/src/consumed-thing.ts

```diff
@@ -371,7 +371,7 @@ export default class ConsumedThing extends TD.Thing implements IConsumedThing {
         this.events = {};
         // Deep clone the Thing Model
         // without functions or methods
-        const clonedModel = JSON.parse(JSON.stringify(thingModel));
+        const clonedModel = structuredClone(thingModel);
```

Comment:
I was wondering how `structuredClone` handles functions on objects and I think it fails.. at least in my simple test it fails with 
`[ERR]: The object could not be cloned.`

Are we sure that there a no functions involved in all cases?
Apparently not since all tests fail but I would like to "ask" the question ;-)

Reply:
There are a few places where I ran into the same error, so I limited the use of `structuredClone` to cases where "pure" objects without functions/methods are being used. So, especially for classes, this new function is apparently not really usable :/ But for regular objects, this new function is recommended over the "traditional" `JSON.parse(JSON.stringify(foo))` approach, as far as I've read.

Reply:
I see, the only thing that bothers me was the comment in the code
`// without functions or methods`

If there are no such "functions" we should remove that comment.
What do you think?

Reply:
Very good point, I made some adjustments in 33af731e4a8a988414155718addc4702e05dd029 :) I also baked the deep cloning documentation into the variable names in these cases, getting rid of the `// Deep clone the Thing Model` comment :)